conduit-extra/Data/Conduit/Text.hs

```diff
@@ -61,7 +60,7 @@ import Data.Streaming.Text
 --
 -- Since 0.3.0
 data Codec = Codec
-    { codecName :: T.Text
+    { _codecName :: T.Text
```

Comment:
I'm not sure why this is being defined in the first place if it's not being used, renaming to an underscore prefix silences the warnings but is quite unsightly. Happy for alternate suggestions on how this should be handled.